netjsonconfig/backends/raspbian/schema.py

```diff
@@ -0,0 +1,4 @@
+from ...schema import schema as default_schema
+from ...utils import merge_config
+
+schema = merge_config(default_schema, {})
```

Comment:
with no schema additions we can't validate mandatory data or enforce checks, are you sure this step is not needed?